src/amo/components/HomeHeroBanner/index.js

```diff
@@ -147,7 +147,64 @@ export class HomeHeroBannerBase extends React.Component<Props> {
 
           <p>
             {i18n.gettext(`Simple zoom in/out tool for a close-up view of any
-              web content.`)}
+              web content`)}
+          </p>
+        </HeroSection>
+      ),
+      (
+        <HeroSection
+          key="hero-12"
+          linkTo="/addon/wikiwand-wikipedia-modernized/"
+        >
+          <h3>{i18n.gettext('Wikiwand')}</h3>
+
+          <p>{i18n.gettext('Give Wikipedia a gorgeous makeover')}</p>
+        </HeroSection>
+      ),
+      (
+        <HeroSection
+          key="hero-13"
+          linkTo="/addon/onetab/"
+        >
+          <h3>{i18n.gettext('OneTab')}</h3>
+
+          <p>
+            {i18n.gettext(`Convert your open tabs into a list and save
+              precious memory space`)}
+          </p>
+        </HeroSection>
+      ),
+      (
+        <HeroSection
+          key="hero-14"
+          linkTo="/addon/kindle-it/"
+        >
+          <h3>{i18n.gettext('Push to Kindle')}</h3>
+
+          <p>{i18n.gettext('Send any Web page to your Kindle device')}</p>
```

Comment:
Editorial: do we still capitalise "web"? 🤷‍♂️

Reply:
@tofumatt I personally don't like capitalizing "Web" but that is how it's referenced in the Firefox copy style guide.

https://docs.google.com/document/d/1zbKQ9vIPDXYm-7G6-MZe3sSmj9yG8Lmb0iwqGku39KM/edit